packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -2278,58 +2289,70 @@ export class Grapher
         this.annotation = annotation
     }
 
-    render(): JSX.Element | undefined {
-        const { isExportingtoSvgOrPng, isPortrait } = this
-        // TODO how to handle errors in exports?
-        // TODO tidy this up
-        if (isExportingtoSvgOrPng) return this.renderPrimaryTab() // todo: remove this? should have a simple toStaticSVG for importing.
-
-        const { renderWidth, renderHeight } = this
+    private renderGrapherComponent(): JSX.Element {
+        const containerClasses = classnames({
+            GrapherComponent: true,
+            GrapherPortraitClass: this.isPortrait,
+            isExportingToSvgOrPng: this.isExportingtoSvgOrPng,
+            GrapherComponentNarrow: this.isNarrow,
+            GrapherComponentSmall: this.isSmall,
+            GrapherComponentMedium: this.isMedium,
+        })
 
-        const style = {
-            width: renderWidth,
-            height: renderHeight,
-            fontSize: this.baseFontSize,
+        const containerStyle = {
+            width: this.renderWidth,
+            height: this.renderHeight,
+            fontSize: this.isExportingtoSvgOrPng
+                ? 18
+                : Math.min(16, this.baseFontSize), // cap font size at 16px
```

Comment:
Note for the reviewers: This has an effect on text in modals only, since text around the chart (header, footer) ignore the base font size and only honour `grapher.isNarrow/isSmall/isMedium`, and the CaptionedChart's `<svg />` element has its own base font size set (i.e. axis labels and chart ticks are only effected by the font size set on the svg element). In modals, I don't want to scale up the font size, that's why it's capped here.